frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/useInitialAutoLayout.ts

```diff
@@ -29,21 +33,25 @@ export const useInitialAutoLayout = () => {
   const { handleLayout } = useAutoLayout()
 
   useEffect(() => {
-    if (initializeComplete) {
-      return
+    const initialize = async () => {
+      if (initializeComplete) {
+        return
+      }
+
+      const tableNameFromUrl = getActiveTableNameFromUrl()
+      updateActiveTableName(tableNameFromUrl)
+      const hiddenNodeIds = await getHiddenNodeIdsFromUrl()
+      addHiddenNodeIds(hiddenNodeIds)
```

Comment:
While the `active` parameter remains in plain text, I noticed that the `hidden` parameter is serialized in base64. As we anticipate an increase in the variety of parameters in the future, it might be worth considering serializing everything in base64, including the keys. However, for the time being, the current specification seems reasonable!